### ISSUE TITLE
Add alt text for images

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#616]
**Problem Description:**
Alt texts are not available on images and cause accessibility issues.

**Expected Behavior/Solution:**
Add alt text to images across LUX pages. The alt text for images that appear on entity pages and that are provided by Wikidata should have the title provided by the copyright statement or the name of the entity. For thumbnail images on the results snippets, the title of the entity should be used as the alt text.

**Requirements:**
- [x] Alt txt must be added to all pages across LUX. 

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] Wireframe/Mockup - Mike
- [x] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [x] Questions
- Where is the alt txt data coming from? 

**UAT/LUX Examples:**
- https://lux-front-dev.collections.yale.edu/view/results/objects?q=%7B%22text%22%3A%22o%27keeffe%22%2C%22_lang%22%3A%22en%22%7D&sq=o%27keeffe

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A 

**Related links:**
- Bugherd: https://www.bugherd.com/projects/284041/tasks/2891

**Wireframe/Mockup:**
N/A as this will aid users that use screen readers.

**Comment by roamye** [comment]
@kamerynB - unsure if there will be a visible change to the FE. I believe not, but I left the mockup section blank in case it does change on the interface.

**Comment by kamerynB** [comment]
@roamye I already confirmed the requirements with the accessibility team. The Expected Behavior is reflective of that. There is no change on the frontend that is visible. This helps screen readers describe the image to users.
